Cargo.lock

```diff
@@ -611,31 +602,11 @@ dependencies = [
  "winapi-util",
 ]
 
-[[package]]
-name = "const-random"
-version = "0.1.8"
-source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "2f1af9ac737b2dd2d577701e59fd09ba34822f6f2ebdb30a7647405d9e55e16a"
-dependencies = [
- "const-random-macro",
- "proc-macro-hack",
-]
-
-[[package]]
-name = "const-random-macro"
-version = "0.1.8"
-source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "25e4c606eb459dd29f7c57b2e0879f2b6f14ee130918c2b78ccb58a9624e6c7a"
-dependencies = [
- "getrandom 0.1.14",
- "proc-macro-hack",
-]
-
 [[package]]
 name = "const_fn"
-version = "0.4.5"
+version = "0.4.4"
```

Comment:
I wonder why downgrading is needed?

Reply:
hmm yeah good eye! I don't know how that snuck in there, I regenerated the file and it now has the `0.4.5` version

Reply:
Thanks for fixing. :)

As a tip, you can just directly edit Cargo.lock as you like to tweak these tiny modifications by hand. I added specific ci check so that you're safe as long as CI says green.